File: ui/src/app/dto/handler-response/customer/customer-command.response.ts
import { CustomerResponseDto } from "../../customer/customer-response-dto";
import { CommandResponse } from "../../response/command-response";

export class CustomerCommandResponse extends CommandResponse {
    customer: CustomerResponseDto;
    token: string;
    constructor(message?: string, error?: boolean, validationErrors?: string[]) {
        super(message, error, validationErrors);
        this.customer = new CustomerResponseDto();
        this.token = '';
    }
}

File: api/Bookshop.Application/Features/Customers/CustomerCommandResponse.cs
using Bookshop.Application.Features.Response;

namespace Bookshop.Application.Features.Customers
{
    public class CustomerCommandResponse : CommandResponse
    {
        public CustomerCommandResponse() : base()
        {

        }
        public CustomerResponseDto? Customer { get; set; }
        public string? Token { get; set; }
    }
}
